h5py/tests/common.py

```diff
@@ -20,11 +20,11 @@
 import numpy as np
 import h5py
 
-if sys.version_info[:2] == (2, 6):
+if sys.version_info[0] == 2:
```

Comment:
Why is this change needed?

Reply:
Ah ok. Was not aware that new Python 3 behavior was also being added to `unittest`. Was under the impression that it was only Python 2.7 backports to older Python versions. Thanks for the pointer.